<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseSettingsWidget</class>
 <widget class="QWidget" name="DatabaseSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>505</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="dbNameLabel">
       <property name="text">
        <string>Database name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="dbNameEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="dbDescriptionLabel">
       <property name="text">
        <string>Database description:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="dbDescriptionEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="defaultUsernameLabel">
       <property name="text">
        <string>Default username:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="defaultUsernameEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="useRecycleBinLabel">
       <property name="text">
        <string>Use recycle bin:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="recycleBinEnabledCheckBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="historyMaxItemsCheckBox">
       <property name="text">
        <string>Max. history items:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="historyMaxItemsSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>2000000000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="historyMaxSizeCheckBox">
       <property name="text">
        <string>Max. history size:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="historyMaxSizeSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="suffix">
          <string> MiB</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2000000000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="cipherLabel">
       <property name="text">
        <string>Encryption cipher:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="cipherCombo"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="kdfLabel">
       <property name="text">
        <string>Key derivation function:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="kdfCombo"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="transformRoundsLabel">
       <property name="text">
        <string>Transform rounds:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSpinBox" name="transformRoundsSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2147483647</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="transformBenchmarkButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Benchmark</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="10" column="1">
      <widget class="QSpinBox" name="memoryCostSpinBox">
       <property name="minimum">
        <number>8192</number>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
       <property name="value">
        <number>1048576</number>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="memoryCostLabel">
       <property name="text">
        <string>Memory cost:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="lanesLabel">
       <property name="text">
        <string>Lanes:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QSpinBox" name="lanesSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dbNameEdit</tabstop>
  <tabstop>dbDescriptionEdit</tabstop>
  <tabstop>transformRoundsSpinBox</tabstop>
  <tabstop>transformBenchmarkButton</tabstop>
  <tabstop>defaultUsernameEdit</tabstop>
  <tabstop>recycleBinEnabledCheckBox</tabstop>
  <tabstop>historyMaxItemsCheckBox</tabstop>
  <tabstop>historyMaxItemsSpinBox</tabstop>
  <tabstop>historyMaxSizeCheckBox</tabstop>
  <tabstop>historyMaxSizeSpinBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
